cvat/requirements/base.txt

```diff
@@ -38,3 +38,4 @@ pascal_voc_writer==0.1.4
 cython==0.29.13
 matplotlib==3.0.3
 scikit-image>=0.14.0
+tensorflow-gpu==1.14.0
```

Comment:
Could you please confirm that it will work without GPU and Nvidia driver?

https://stackoverflow.com/questions/43558707/tensorflow-importerror-libcusolver-so-8-0-cannot-open-shared-object-file-no